pkg/kubelet/config/file_test.go

```diff
@@ -36,7 +38,7 @@ import (
 func TestExtractFromNonExistentFile(t *testing.T) {
 	ch := make(chan interface{}, 1)
 	c := sourceFile{"/some/fake/file", "localhost", ch}
-	err := c.extractFromPath()
+	err := c.watch()
```

Comment:
For testing the extract feature, what is the purpose of changing from extractFromPath() to watch()?